DispatchGroup/ViewController.swift

```diff
@@ -59,7 +59,7 @@ class ViewController: UIViewController {
     
```

Comment:
it is not mandatory to use SDWebImage for fetching images from network. Use simple UrlSession